fds/utils.py

```diff
@@ -134,12 +134,64 @@ def get_input_from_user(question: str, type: str = "input") -> str:
     :param type: The type of input
     :return: output from the user
     """
-    questions = [
-        {
-            'type': type,
-            'message': question,
-            'name': 'question'
-        }
-    ]
-    answers = PyInquirer.prompt(questions)
-    return answers['question']
+    get_input = input
+    if type == "password":
+        get_input = getpass.getpass
+    display_text_to_user = f"{question}:"
+    answer = get_input(display_text_to_user)
+    if answer is None:
+        return get_input_from_user(question, type)
+    return answer
+
+
+def get_expand_input_from_user(question: str, choices: List[Dict[str, str]], default: Union[str, bool, int]) -> str:
+    """
+    Get expand input from the user
+    :param question: Question to ask the user
+    :param choices: List of choices from user
+    :param default: Default selected value
+    :return: user selected value
+    """
+    # Only pick choice keys
+    choice_keys = list(map(lambda x: x["key"], choices))
```

Comment:
I think all these list manipulations could be a made a lot simpler if you just turn `choices` into `Dict[str,Dict[str,str]]` with the first key being `x["key"]`.

Reply:
Yeah I thought about it, but preserving the inputs of pyinquirier for now, I'll do this in a later PR if that makes sense.

Reply:
You can preserve the inputs and just have one line at the start of this function that does something like
`choices = {x['key'] : x for x in choices)`

Reply:
Fixed